zaza/openstack/charm_tests/glance/tests.py

```diff
@@ -222,3 +223,51 @@ def test_100_create_delete_image(self):
         )
         self.assertEqual(image["size"], response["ContentLength"])
         openstack_utils.delete_image(self.glance_client, image["id"])
+
+
+class GlanceCinderBackendTest(test_utils.OpenStackBaseTest):
+    """Encapsulate glance tests using cinder backend."""
+
+    @classmethod
+    def setUpClass(cls):
+        """Run class setup for running glance tests with cinder backend."""
+        super(GlanceCinderBackendTest, cls).setUpClass()
+        cls.glance_client = openstack_utils.get_glance_session_client(
+            cls.keystone_session
+        )
```

Comment:
I don't think this style of formatting is prevalent in the lib - e.g. trailing ')' on a new line.  I think we normally put it on the tail end of the previous line.

Reply:
Done